src/frontend/src/flows/about/index.ts

```diff
@@ -2,115 +2,102 @@ import { html, render } from "lit-html";
-    <p class="t-paragraph">
-      Your Identity Anchor is a number that connects you to your accounts on
-      applications across the Internet Computer. Store your Anchor somewhere
-      safe. If you lose it, you lose access to all of the accounts that you
-      created with it. Internet Identity keeps your anchor private and does not
-      share it with any applications.
-    </p>
-  </div>
-  <div class="l-stack">
-    <h2 class="t-title">Easy and Secure Login</h2>
-    <p class="t-paragraph">
-      Get the privacy of many separate accounts, with the convenience of a
-      single sign-on. Internet Identity generates a new private key for every
-      dapp that you use, and stores the keys on your device. With just your
-      Identity Anchor and device, you can manage hundreds of private keys and
-      accounts.
-    </p>
-  </div>
-  <div class="l-stack">
-    <h2 class="t-title">Explore dapps on the Internet Computer</h2>
-    <p class="t-paragraph">
-      With an Identity Anchor, you can create accounts with dapps on the
-      Internet Computer, like:
-    </p>
-    <ul class="c-list c-list--bulleted l-stack">
-      <li class="c-list__item">
-        <a
-          href="https://distrikt.app/"
-          target="_blank"
-          rel="noopener noreferrer"
-          >Distrikt</a
-        >
-        <p>
-          A decentralized, professional social media network that empowers users
-          to own and control their identity and data.
-        </p>
-      </li>
-      <li class="c-list__item">
-        <a
-          href="https://dscvr.ic0.app/"
-          target="_blank"
-          rel="noopener noreferrer"
-          >Discovr</a
-        >
-        <p>
-          A decentralized social media platform that rewards users for their
-          contributions.
-        </p>
-      </li>
-      <li class="c-list__item">
-        <a href="https://oc.app/" target="_blank" rel="noopener noreferrer">
-          OpenChat
-        </a>
-        <p>
-          OpenChat is a fully featured chat application running end-to-end on
-          the Internet Computer blockchain.
-        </p>
-      </li>
-    </ul>
-  </div>
-  <div class="l-stack">
-    <h2 class="t-title">Compatibility</h2>
+import copyJson from "./copy.json" assert { type: "json" };
 
-    <div class="l-horizontal l-stack">
-      <div class="about__compatibility-flexchild">
-        <h3 class="t-title">Recommended Desktop</h3>
-        <ul class="c-list c-list--bulleted l-stack">
-          ${compatibilityData.desktop.map(
-            (browser) => html`<li>${browser}</li>`
-          )}
-        </ul>
-      </div>
-      <div class="about__compatibility-flexchild">
-        <h3 class="t-title">Recommended Mobile</h3>
-        <ul class="c-list c-list--bulleted l-stack">
-          ${compatibilityData.mobile.map(
-            (browser) => html`<li>${browser}</li>`
-          )}
-        </ul>
+const aboutContentSlot = (i18n: I18n) => {
```

Comment:
nth: because I like to descruct everything, would following work and be cool?

```
const aboutContentSlot = ({i18n}: I18n) => {
    const copy = i18n(copyJson);
```

Reply:
For consistency I'll keep "1 arg => inline, 2+ => destruct"